pages/citing.md

```diff
@@ -0,0 +1,32 @@
+layout: page
+title: Citing
+permalink: /citing
+nav_order: 98
+---
+
+
+Citing Scikit-HEP or individual packages {#citing}
+========================================
+
+We appreciate if you consider to acknowledge the Scikit-HEP project in case it has made or is having an impact on your work or research.
+We suggest the following citation for the project as a whole (in BibTeX format):
+
+```
```

Comment:
```suggestion
```BibTeX
```

Reply:
Thanks Henry. Funny enough I was looking for something like that but did not find it from a quick search.